beacon_node/execution_layer/src/payload_status.rs

```diff
@@ -55,22 +57,10 @@ pub fn process_payload_status(
                     })
                 }
             }
-            PayloadStatusV1Status::Invalid => {
-                if let Some(latest_valid_hash) = response.latest_valid_hash {
-                    // The response is only valid if `latest_valid_hash` is not `null`.
-                    Ok(PayloadStatus::Invalid {
-                        latest_valid_hash,
-                        validation_error: response.validation_error.clone(),
-                    })
-                } else {
-                    Err(EngineError::Api {
-                        error: ApiError::BadResponse(
-                            "new_payload: response.status = INVALID but null latest_valid_hash"
-                                .to_string(),
-                        ),
-                    })
-                }
-            }
+            PayloadStatusV1Status::Invalid => Ok(PayloadStatus::Invalid {
+                latest_valid_hash: response.latest_valid_hash,
+                validation_error: response.validation_error.clone(),
```

Comment:
We can remove this clone, not sure why Clippy didn't get it.

```suggestion
                validation_error: response.validation_error,
```